src/scss/components/_aside.scss

```diff
@@ -26,12 +26,6 @@ aside {
 					width: rem(158);
 					word-wrap: break-word;
 
-					&:active {
```

Comment:
Personally I think it would be better to keep this section and remove the one down below because this section stays within `li` block and more impact.

Reply:
Functionally that currently does not work. Lines 29-33 are used to bold the aside bar on click. Lines 48-52 are used to bold the aside bar on scroll.

Reply:
It'll work if you change the colon (:) in the line 29 to a dot (.): `&:active` -> `&.active`

Reply:
Yes, it works now! Thanks @cindyli